contrib/python-flask/template.py

```diff
@@ -0,0 +1,35 @@
+pkgname = "python-flask"
+pkgver = "3.0.1"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-flit_core",
+    "python-installer",
+    "python-wheel",
+]
+checkdepends = [
+    "python-blinker",
+    "python-click",
+    "python-itsdangerous",
+    "python-jinja2",
+    "python-pytest",
+    "python-werkzeug",
+]
+depends = [
+    "python-blinker",
+    "python-click",
+    "python-itsdangerous",
+    "python-jinja2",
+    "python-werkzeug",
+]
```

Comment:
put depends above checkdepends then to `checkdepends = ["python-pytest"] + depends`

Reply:
fixed